### DISCUSSION TITLE
Sx1262+Nucleo-F746ZG does lorawan works with the provided lorawan library

### DISCUSSION THREAD

**Discussion by KrithikaPhyavula** [jgromes/RadioLib#1125] [Q&A]
Hi, I have worked on Lora with the sx1276 with nucleo. And also worked on Nucleo-L series boards with lorawan. But I can't find or generate some working code with the nucleo-f7 combined with the sx1262 or sx1276. I am struggling to check all the available and existing examples and failing. Can anyone please support me in finding some useful resource. My aim to exist the lorawan communication with the nucleo-f746zg with sx1262 and connecting it to the aws or ttn.

**Comment by jgromes** [comment]
>  I am struggling to check all the available and existing examples and failin

I suggest checking out the SX126x examples and getting your device to transmit just plain LoRa packets. This will help you check basic things like pinout. Then move on to the LoRaWAN examples for details on setting up LoRaWAN-specific stuff like device IDs and keys. However, don't expect someone will just post a working code for you. We don't know your setup nor your hardware.

**Reply by KrithikaPhyavula** [reply]
I have worked on lora with sx1276 and lorawan communication by using the stm32l4 board with sx1262 it worked with the help of I-cube-lrwan software provided by stm community but to make it work with stm32f7 it became too tough to shift all the peripherals from the available example. I thought of asking whether someone worked on establishing lorawan with nucleo boards rather than nucleo-l series . Thanks for your help

**Reply by jgromes** [reply]
I've read all of this in your original post. My point is the following: there are literally dozens of Nucleo boards, and dozens of SX1262 modules. It is extremely unlikely someone else has the exact same Nucleo board, radio module, wiring and use case as you have. So in order for anybody to be able to help you, you will have to post what you tried (the code, description of your setup, which hardware you use etc.), and what was the result.

> I thought of asking whether someone worked on establishing lorawan with nucleo boards rather than nucleo-l series

Most likely, in RadioLib there is no functional difference between LoRaWAN on Nucleo or any other platform, like ESP32. The only thing you have to do is provide the correct pinout. But nobody apart from you knows that pinout, hence why we can't help you.
